scen_def/SXC_Adventure.cfg

```diff
@@ -110,19 +110,22 @@
     {SXC_ENEMY_LEADER_SET_SPECIALS 6 2 append {WEAPON_SPECIAL_DRAIN} {BLANK}}
     {SXC_ENEMY_LEADER_SET_SPECIALS 6 3 append {WEAPON_SPECIAL_CHARGE} {BLANK}}
     {SXC_ENEMY_LEADER_ADD_ABILITY 6 {ABILITY_SXC_DARKAURA}}
+    {SXC_ENEMY_LEADER_HALO 6 {SXC_DARKAURA_HALO}}
```

Comment:
I'd like to split this PR in to multiple parts:
* one for adding halos to bosses that already have them (so it's just a visual hint of the existing stats)
* one or more for changing the stats on the bosses, that would need map-by-map playtesting